service-broker/src/main/java/io/enmasse/osb/api/bind/OSBBindingService.java

```diff
@@ -78,6 +78,7 @@ public Response bindServiceInstance(@Context SecurityContext securityContext, @P
             this.random.nextBytes(passwordBytes);
             String password = Base64.getEncoder().encodeToString(passwordBytes);
 
+            log.info("User: {}, password; {}", username, password);
```

Comment:
I know that OpenShift secrets are not exactly secure, but I still think we should avoid logging passwords

Reply:
@rgodfrey I think @lulf log this info only in for debugging this issue :) It will be removed after debugging.

Reply:
@rgodfrey Yes, this is not intended to be merged.